be/src/common/daemon.cpp

```diff
@@ -230,9 +231,13 @@ void Daemon::memory_gc_thread() {
         if (!MemInfo::initialized() || !ExecEnv::GetInstance()->initialized()) {
             continue;
         }
+        auto sys_mem_available = doris::MemInfo::sys_mem_available();
+
```

Comment:
Also need to save `doris::MemInfo::proc_mem_no_allocator_cache()`, minus the memory size released by Group gc

Reply:
done